### ISSUE TITLE
[Feature]  Does this service work like a CDN, distributed across global regions when hosted via fly.io ?

### ISSUE THREAD

**Issue by honeyhead** [kevinanielsen/go-fast-cdn#151]
What I'm curious about will be the cost of CDN.
I think there's a cost for data transmission and reception on fly.io , but there's also storage
How about supporting external S3 with storage?
For example, Cloudflare R2 can certainly reduce operational costs when using storage for CDN purposes.

**Comment by kevinanielsen** [comment]
No, this should be considered more like a simple file-hosting platform :)
